rust/operator-binary/src/main.rs

```diff
@@ -74,7 +75,9 @@ async fn main() -> anyhow::Result<()> {
                 watch_namespace.get_api::<TrinoCluster>(&client),
                 watcher::Config::default(),
             );
-            let cluster_store = cluster_controller.store();
+            let cluster_store_1 = Arc::new(cluster_controller.store());
+            let cluster_store_2 = cluster_store_1.clone();
```

Comment:
Name it `catalog_cluster_store` and `authentication_class_cluster_store`?

Reply:
Honestly I don't care :shrug: